app/src/test/java/com/example/vybe/VibeEventTest.java

```diff
@@ -58,16 +58,12 @@ public void VibeEventConstructor_FullInit() {
         assertEquals(testVibe.getName(), testVibeEvent.getVibe().getName());
         assertEquals("just really happy", testVibeEvent.getReason());
         assertEquals(testDate.getClass(), testVibeEvent.getDateTime().getClass());
+        assertEquals(formatter.format(testDate), testVibeEvent.getDateTimeString());
         assertEquals("Alone", testVibeEvent.getSocSit().toString());
         assertEquals("", testVibeEvent.getId());
         assertEquals("image", testVibeEvent.getImage());

```

Comment:
In the previous empty constructor test, is it now possible to try and `assertNull` on the latitude and longitude now that they are `Double`'s?

Reply:
Yes it is! I just updated the constructer tests to check for this